.mailmap

```diff
@@ -0,0 +1,12 @@
+# This file teaches `git log` and friends the canonical names
+# and email addresses to use for our contributors.
+#
+# See https://github.com/zulip/zulip/blob/main/.mailmap for Zulip's
+# documentation on this file.
+
```

Comment:
This doesn't quite combine all Sumanth's contributions.

Otherwise, we definitely would benefit from one for @Ezio-Sarthak (commits are split fairly evenly), and perhaps yourself after this commit :)